### PR TITLE
Add Python 3.7 support.

### PR DESCRIPTION
**Pull request by icemac**
And do some clean-up.

Closes #6.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,16 +2,19 @@ language: python
 sudo: false
```
Comment (mgedmin) [inline .travis.yml]: You might want to drop this while cleaning things up.

File: .travis.yml [hunk 2]
```diff
@@ -2,16 +2,19 @@ language: python
...
 python:
     - 2.7
-    - pypy-5.6.0
+    - pypy
```
Comment (mgedmin) [inline .travis.yml:4]: Do we want to add pypy3?

**Review by mgedmin — APPROVED** [review]
LGTM

File: .travis.yml (hunk 2 above)
Comment (icemac) [reply .travis.yml:4]: Let's try it, in #6 the interpreter crashed during the tests.

Comment (mgedmin) [reply .travis.yml:4]: Okay, let's drop it then.

pypy3 is alpha for a reason, and "Segmentation fault" could very well be that reason.

**Comment by icemac** [comment]
Removed PyPy3 trails in force push.

**Pull request merged**